src/App.vue

```diff
@@ -1,14 +1,22 @@
 <template>
   <div id="nav">
-    <router-link to="/">Home</router-link> |
-    <router-link v-if="!isLoggedIn" to="/login/">Login</router-link>
-    <a href="#" v-if="isLoggedIn" @click="logout"> Logout</a>
+    <router-link to="/">{{ $t("nav.home") }}</router-link> |
+    <router-link v-if="!isLoggedIn" to="/login/">{{ $t("nav.login") }}</router-link>
+    <a href="#" v-if="isLoggedIn" @click="logout">{{ $t("nav.logout") }}</a>
   </div>
+  <LocaleSwitcher></LocaleSwitcher>
   <router-view />
 </template>
 
 <script>
+import LocaleSwitcher from "./components/LocaleSwitcher.vue";
 export default {
+  components: {
+    LocaleSwitcher,
+  },
+  created() {
+    console.log(this.$i18n.availableLocales);
+  },
```

Comment:
can remove this if it was just used for local testing?
The console logs are not used when deploying on prod.
If a developer needs to see this info in console, then we can let it be
otherwise remove it

Reply:
Deepansh -- didn't you enable some plugin that removes all console logs before shipping? Or am I remembering wrong?

Reply:
Yes yes it does remove
But only when we compile for deployment
While development, the console logs will remain

I try to remove them after developing and testing because they clog up the chrome console (in local dev)

None of this will matter when shipping

Reply:
Removed this!